rand_seeder/src/sip.rs

```diff
@@ -372,4 +457,45 @@ mod test {
             assert_eq!(hasher.finish(), *val, "mismatch with reference vector for i={}", *len);
         }
     }
+    
+    #[test]
+    fn test_rng_vectors() {
+        // SipRng has no external reference. These vectors are defined here.
+        
+        let mut seed = [0u8; 32];
+        let mut rng = SipRng::from_seed(seed);
+        
+        let vector = [0x4c022e4ec04e602a, 0xc2c0399c269058d6,
+            0xf5c7399cde9c362c, 0x37e5b9491363680a,
+            0x9582782644903316, 0x2a9d2e160aad88d,
+            0x983958db9376e6f6, 0xdead8960b8524928,
+            0xcfa886c6642c1b2f, 0x8f8f91fcf7045f2a,
+            0x1bbda585fc387fb3, 0x242485d9cc54c688,
+            0x9be110f767d8cee, 0xd61076dfc3569ab3,
+            0x8f6092dd2692af57, 0xbdf362ab8e29260b];
+        // for _ in 0..8 {
+        //     println!("0x{:x}, 0x{:x},", rng.next_u64(), rng.next_u64());
+        // }
```

Comment:
You can probably leave uncomment this code. The output will only be shown if the test fails.

Reply:
In theory it makes the tests slower to run.